zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -4728,6 +4727,9 @@ components:
           type: integer
           format: int64
           description: The key of the decision instance.
```

Comment:
💭 Make clear here that the key is *not* a unique PK for this entity (in contrast to all other entities)?